sdk/translation/Azure.AI.Translation.Document/CHANGELOG.md

```diff
@@ -1,14 +1,28 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 1.1.0-beta.1 (2024-05-07)
 
 ### Features Added
+- Added Synchronous document translation API: {RESOURCE}/translator/document:translate?api-version=[Date]
```

Comment:
Since this is on the SDK package it would be nicer to mention the SDK method here.

Reply:
Added